src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py

```diff
@@ -284,8 +287,7 @@ def get_op_type(self):
             return 'UNKNOWN_TYPE_' + str(self.pt_module.op)
 
     def get_schema(self):
-        return ''
-        return self.pt_module.schema()
+        return 'NONE'
```

Comment:
strange function:)

Reply:
It is useful in TS decoder, for FX it doesn't have meaning

Reply:
what about standard `None` value?

Reply:
We need string since that is function that is used from c++ code